File: Project/Java/src/eshop/Paper.java
package eshop;

public class Paper extends Item{
    private int weight;
    private int pages;
    private static int count = 0;


    Paper(String name, double price, String description, int stock, int id, int weight, int pages){
        setName(name);
        setPrice(price);
        setDescription(description);
        setStock(stock);
        this.weight = weight;
        this.pages = pages;
        count++;
    }
    public String getDetails(){
        return "Weight: " + weight + ", Pages: " + pages;
    }

    public static int getCount() {
        return count;
    }
}

File: Project/C++/Project1/Paper.cpp
#include "Paper.h"
int Paper::ammount = 0;
Paper::Paper(std::string name, int price, std::string description, int stock, int id, int weight, int pages)
{
	setName(name);
	setPrice(price);
	setDescription(description);
	setStock(stock);
	setId(id);
	setObjType("paper");
	this->weight = weight;
	this->pages = pages;
	this->ammount++;
}

std::string Paper::getDetails()
{
	return "Weight: " + std::to_string(weight) + ". Pages: " + std::to_string(pages) + ".";
}
